app/src/controllers/user/sagas.js

```diff
@@ -105,6 +107,21 @@ function* fetchUserWorker() {
     user = yield call(fetch, URLS.user());
     yield put(fetchUserSuccessAction(user));
   } catch (err) {
+    const error = err.message;
+    if (error === ERROR_UNAUTHORIZED) {
+      yield put(
+        showNotification({
```

Comment:
1. Message should not be shown when user starts on login page without redirecting on token expiration.
2. Redirecting on token expiration has been already implemented (see `initAuthInterceptor` in `common/utils/fetch`). Have you investigated the reason why it doesn't work in this case?

Reply:
Fixed.